common/ast/src/ast/statement.rs

```diff
@@ -351,6 +374,60 @@ pub enum DatabaseEngine {
     Github(String),
 }
 
+/// CopyTarget is the target that can be used in `COPY INTO`.
+#[derive(Debug, Clone, PartialEq)]
+pub enum CopyTarget<'a> {
+    /// Table can be used in `INTO` or `FROM`.
+    ///
+    /// While table used as `FROM`, it will be rewrite as `(SELECT * FROM table)`
+    Table(
+        Option<Identifier<'a>>,
+        Option<Identifier<'a>>,
+        Identifier<'a>,
+    ),
```

Comment:
This may make its purpose clearer.

```suggestion
    Table {
        catalog: Option<Identifier<'a>>,
        database: Option<Identifier<'a>>,
        table: Identifier<'a>,
    },
```